src/app/containers/MediaPlayer/Metadata.jsx

```diff
@@ -14,4 +15,10 @@ const Metadata = aresMediaBlock => {
   );
 };
 
+Metadata.propTypes = {
+  aresMediaBlock: shape({
+    model: shape({}),
```

Comment:
Is there any way we can be more specific with what we expect the `model` object to look like?